### PR TITLE
V6.00 Final Release PR

### PR DESCRIPTION
**Pull request by x86fr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,8 +108,10 @@ For test purposes, there is also an option to build an ISO image that uses
 GRUB as an intermediate bootloader. See the `Makefile` in the `build32` or
 `build64` directory for details. The ISO image is both legacy and UEFI
 bootable, so you need GRUB modules for both legacy and EFI boot installed
-on your build system. You may need to adjust some path and file names in
-the make file to match the naming on your system.
+on your build system (e.g. on Debian, the required GRUB modules are located
+in packages `grub-pc-bin`, `grub-efi-ia32-bin` and `grub-efi-amd64-bin`).
+You may need to adjust some path and file names in the make file to match
```
Comment (debrouxl) [inline README.md:113]: "in the make file" is correct, as it is a file for `make`, but "in the Makefile" would probably be a more usual wording ?

File: README.md [hunk 2]
```diff
@@ -33,15 +35,15 @@ later 32-bit or 64-bit CPU.
...
 
   * make the code more readable and easier to maintain
   * make the code 64-bit clean and support UEFI boot
   * fix failures seen when building with newer versions of GCC
 
-In the process of creating PCMemTest, a number of features of Memtest86+ v5.01
+In the process of creating PCMemTest, a number of features of Memtest86+ v5
 that were not strictly required for testing the system memory were dropped. In
 particular, no attempt was made to measure the cache and main memory speed, or
 to identify and report the DRAM type. These features were added back in
```
Comment (debrouxl) [inline README.md:49]: "were added back and expanded" ?

File: README.md [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
+restricted by the memory used by the operating system and not depending
+on any underlying software like UEFI libraries.
 
 Memtest86+ can be loaded and run either directly by a PC BIOS (legacy or UEFI)
 or via an intermediate bootloader that supports the Linux 16-bit, 32-bit,
 64-bit, or EFI handover boot protocol. It should work on any Pentium class or
 later 32-bit or 64-bit CPU.
 
+Binary releases (both stable and nightly dev builds) are available on
+[memtest.org](https://memtest.org).
```
Comment (martinwhitaker) [inline README.md:17]: although the nightly build is still stuck at commit ddbee66 :-(

**Review by martinwhitaker — APPROVED** [review]
Good to go in my opinion.

File: README.md (hunk 3 above)
Comment (x86fr) [reply README.md:17]: The Windows Server VM used to build the nightly windows USB installer will be back online tomorrow :)

Comment (x86fr) [reply README.md:17]: Nighty Build server is back online.

**Pull request merged**